src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['audit_log.pipe'] = format_text([[
+    Specify a pipe for the audit log destination. You can set the `pipe` type
+    using the audit_log.to option. If log is a program, its pid is stored in
+    the `audit.pid` field. You need to send it a signal to rotate logs.
+]])
+
+M['audit_log.spaces'] = format_text([[
+    The array of space names for which data operation events (`space_select`,
+    `space_insert`, `space_replace`, `space_delete`) should be logged. The array
+    accepts string values. If set to box.NULL, the data operation events are
+    logged for all spaces.
+]])
+
+M['audit_log.spaces.*'] = format_text([[
+    A specific space name in the array for which data operation events are
+    logged. Each entry must be a string representing the name of the space
+    to monitor.
+
+    Example:
+
+    `spaces: [bands, singers]`, only the events of `bands` and `singers` spaces
+    are logged.
+]])
+
+M['audit_log.to'] = 'Enable audit logging and define the log location.'
+
+M['audit_log.syslog'] = format_text([[
+    This module allows configuring the system logger (syslog) for audit
+    logs in Tarantool. It provides options for specifying the syslog server,
+    facility, and identity for logging messages.
+]])
+
+M['audit_log.syslog.facility'] = format_text([[
+    Specify a system logger keyword that tells syslogd where to send the
+    message. You can enable logging to a system logger using the
+    `audit_log.to` option.
+]])
+
+M['audit_log.syslog.identity'] = format_text([[
+    Specify an application name to show in logs. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+M['audit_log.syslog.server'] = format_text([[
+    Set a location for the syslog server. It can be a Unix socket path
+    starting with "unix:" or an ipv4 port number. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+-- }}} audit_log configuration
+
+-- {{{ compat configuration
+
+M['compat'] = format_text([[
+    The `compat` section defines values of the compat module options.
+]])
+
+M['compat.binary_data_decoding'] = format_text([[
+    Define how to store binary data fields in Lua after decoding:
+
+    - `new`: as varbinary objects
+    - `old`: as plain strings
+]])
+
+M['compat.box_cfg_replication_sync_timeout'] = format_text([[
+    Set a default replication sync timeout:
+
+    - `new`: 0
+    - `old`: 300 seconds
+]])
+
+M['compat.box_consider_system_spaces_synchronous'] = format_text([[
+    Controls whether to consider system spaces synchronous when the
+    synchronous queue is claimed, regardless of the user-provided `is_sync`
+    option. Either enables synchronous replication for system spaces when the
+    synchronous queue is claimed, overriding the user-provided 'is_sync' space
+    option, or falls back to the user-provided 'is_sync' space option.
+]])
```

Comment:
Compat options above have explicit old/new behavior descriptions and it is useful when you're out of context of particular changes in tarantool. Can we add them here too?

Reply:
Added.
```lua
I['compat.box_consider_system_spaces_synchronous'] = format_text([[
    Whether to consider most system spaces as synchronized regardless
    of the `is_sync` space option:

    - `new` (4.x default): System spaces are synchronized when
      the synchronous queue is claimed (`box.info.synchro.queue.owner ~= 0`),
      exceptfor `vinyl_defer_delete` (local space) and `sequence_data`
      (synchronized by user spaces)
    - `old` (3.x default): System spaces are not synchronized unless
      explicitly marked with the `is_sync` option
]])
```

Reply:
exceptfor -> except for

> `sequence_data` (synchronized by user spaces)

I can't understand this senstence, primarily because I can't match 'synchronized by a space' to anything known to me. Can we catch the wording from 371fa0dc0f49a00bcc839b6a1976884a5d12fd80 (where it is synchorized by a space operation, not by a space)?

Otherwise, OK.

Reply:
`(synchronized by user spaces)` replaced with `(synchronized by synchronous user spaces operations)`.